contrib/capitalone_dataprofiler_expectations/capitalone_dataprofiler_expectations/tests/expectations/metrics/test_core.py

```diff
@@ -214,7 +214,7 @@ def test_data_profiler_column_list_metric_same_as_profile_report_pd(
     table_columns_metric: MetricConfiguration
     results: dict
 
-    table_columns_metric, results = get_table_columns_metric(engine=engine)
+    table_columns_metric, results = get_table_columns_metric(execution_engine=engine)
```

Comment:
I must say, I really do like this more-explicit name change. I do hope it the additional distinction between sa.engine and execution engine will continue to pay us dividends.